js_sdk/install_glia_with_site_id.js

```diff
@@ -0,0 +1,18 @@
+var installGlia = function(siteId, callback) {
+  var gliaIntegrationScriptUrl = 'https://api.salemove.com/salemove_integration.js?site_id=' + siteId;
+  var scriptElement = document.createElement('script');
+
+  scriptElement.async = 1;
+  scriptElement.src = gliaIntegrationScriptUrl;
+  scriptElement.type = 'text/javascript';
+  if (typeof(callback) === 'function') {
+    scriptElement.addEventListener('load', callback);
+  }
+  
+  document.body.append(scriptElement);
+};
+
+var siteId = '11111111-2222-3333-4444-555555555555'; // This can be requested from your Client Success Manager
```

Comment:
since this is for integrators who know little about. our domain I would either change the comment to 

// This is a X digits UUID for your site which can be requested to your Client Success Manager or via our Service. desk [link]

or 
// This is a UUID for your side. You can read more about it in this support article. 

in. the support article we should explain how they can. get it from the CSM or via Service Desk